attentionbench/attention_utils.py

```diff
@@ -66,8 +67,11 @@ class TuningSpec:
 
     def get_lowering_config(self) -> str:
         return (
-            f"#iree_codegen.lowering_config<"
-            + f"tile_sizes = [[{','.join([str(x) for x in self.wg_tiles])}]]"
+            f"#iree_gpu.lowering_config<"
+            + "{ "
+            + f"workgroup = [{",".join([str(x) for x in self.wg_tiles])}], "
+            + f"reduction = [{",".join(str(x) for x in self.red_tiles)}]"
```

Comment:
You can do `", ".join(map(str, range))` (also notice the space)

Reply:
I think use `'` instead of `"` for CI failure